test/types/string/psahabu/split-whitespace.chpl

```diff
@@ -0,0 +1,51 @@
+use Time;
+
+config const timing = true;
+
+var spaces = "The quick brown fox jumps over the lazy dog";
+var tabs = "The\tquick\tbrown\tfox\tjumps";
+var newLines = "The\nquick\nbrown\nfox\njumps";
+var returns = "The\rquick\rbrown\rfox\rjumps";
+var bigSpace = "The quick brown           fox   jumps";
+var whitespace = " \t\r\n";
+var empty = "";
+
```

Comment:
I think it would be good to have some strings with mixed characters (i.e., "\t   \r\n\t")